src/ucm/util/log.c

```diff
@@ -168,7 +170,18 @@ static void ucm_log_vsnprintf(char *buf, size_t max, const char *fmt, va_list ap
                 if (!value.s) {
                     value.s = "(null)";
                 }
+#if defined(__riscv)    
+               trace_idx = strstr(value.s, "TRACE") - value.s;
+               if(trace_idx > 0) {
+                    pad -= strnlen(value.s, trace_idx+6);
+               }
+               else if (trace_idx == 0) {
```

Comment:
there are few code style issues, pls see https://github.com/openucx/ucx/wiki/Code-style-checking

Reply:
found and fixed indentation issues. we had errors running the `.clang-format` script.